packages/expo-updates/ios/EXUpdates/EXUpdatesUtils.h

```diff
@@ -18,6 +18,7 @@ NS_ASSUME_NONNULL_BEGIN
 + (NSString *)getRuntimeVersionWithConfig:(EXUpdatesConfig *)config;
 + (NSURL *)urlForBundledAsset:(EXUpdatesAsset *)asset;
 + (NSString *)pathForBundledAsset:(EXUpdatesAsset *)asset;
++ (BOOL)nativeDebugging;
```

Comment:
might be good to add a docblock here briefly explaining why/when this is used.

Reply:
second this. somehow the `nativeDebugging` is not very clear for what it is. maybe something like `enableUpdatesInDebugBuild`. do it sounds better?

Reply:
i just got confused for `EXAppDefines.APP_DEBUG && !EXUpdatesUtils.nativeDebugging()`. how could it happen for a debug build without native debugging. later i realize it means to enable expo-updates autosetup on debug build.